src/style.css

```diff
@@ -0,0 +1,76 @@
+*,
+::after,
+::before {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(Avenir.ttc);
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+.searchbar {
+  position: relative;
+  margin-top: 20px;
+}
+
+.big_bar,
+.small_bar {
+  font-family: inherit;
+  color: #3d4d61;
+  background: #fff;
+
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+}
+
+.big_bar {
+  width: 100%;
+  height: 70px;
+
+  font-size: 16px;
+  padding-left: 62px;
+}
+
+#magnifier_big {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.small_bar {
+  width: 100%;
+  height: 42px;
+
+  font-size: 14px;
+  padding-left: 33px;
+}
+
+#magnifier_small {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  top: 15px;
+}
+
+.hovered:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
```

Comment:
change only what you need(ex: you can omit you do not need hovered class here)

Reply:
ok!